concent-vm/Vagrantfile

```diff
@@ -0,0 +1,87 @@
+Vagrant.configure("2") do |config|
+
+    unless Vagrant.has_plugin?("vagrant-vbguest")
+        puts "Please install `vagrant-vbguest` plugin before execute `vagrant up`."  +
+             "\nRemember to install it in other directory then concent-vm/"
+        abort
+    end
+
+    config.vm.box      = "debian/stretch64"
+    config.vm.hostname = "concent-development-vm"
+    config.vm.synced_folder(".", "/vagrant", disabled: true)
+
+
+    config.vm.network "private_network", ip: "172.40.2.3"
+
+    blockchain_disk = './disk/blockchain_disk.vdi'
+    config.vm.provider "virtualbox" do |virtualbox|
+        virtualbox.memory = 2048
+        virtualbox.gui    = false
+        if not File.exists?(blockchain_disk)
+            virtualbox.customize [
+                'createhd',
+                '--filename', blockchain_disk,
+                '--variant',  'Fixed',
+                '--size',     30 * 1024, # MB
+            ]
+        end
+        virtualbox.customize [
+            'storageattach',
+            :id,
+            '--storagectl',   'SATA Controller',
+            '--port',         1,
+            '--device',       0,
+            '--type',         'hdd',
+            '--medium',       blockchain_disk,
+            '--hotpluggable', 'on'
+        ]
+    end
+
+    if not File.exists?(blockchain_disk)
+        config.vm.provision "shell", privileged: true, inline: <<-SHELL
+            mkfs.ext4 /dev/sdb
+        SHELL
+    end
+
+    config.vm.provision "shell", run: "always", privileged: true, inline: <<-SHELL
+        directory=/home/vagrant/blockchain
+        if [ ! -d "$directory" ]; then
+            mkdir $directory || exit 1
+        fi
+        mountpoint --quiet $directory || mount /dev/sdb $directory
+    SHELL
+
+    def detaching_virtual_disk(trigger, blockchain_disk)
```

Comment:
This does not sound like a function name. The name should sound like a command rather than a statement of fact.
`detaching` -> `detach`.